experimental/devices/inky/inky.go

```diff
@@ -332,14 +329,19 @@ func (d *Dev) sendCommand(command byte, data []byte) error {
 }
 
 func (d *Dev) sendData(data []byte) error {
-	if len(data) > 4096 {
-		return fmt.Errorf("sending more data than chunk size: %d > 4096", len(data))
-	}
 	if err := d.dc.Out(gpio.High); err != nil {
```

Comment:
I wonder what's the required timing in ms between setting the line and sending the actual commands and data. i.e. if a very small sleep should be added.

Reply:
I found the datasheet: https://www.china-epaper.com/uploads/soft/DEPG0420R01V3.0.pdf - it has timings for the 3-wire SPI configuration but not the 4-wire configuration with D/C pin that we're using here :/